.env.example

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_NETWORK=stokenet # Options: mainnet or stokenet
-NEXT_PUBLIC_DAPP_DEFINITION_ADDRESS=account_tdx_2_129kev9w27tsl7qjg0dlyze70kxnlzycs8v2c85kzec40gg8mt73f7y
\ No newline at end of file
+NEXT_PUBLIC_DAPP_DEFINITION_ADDRESS=account_tdx_2_129kev9w27tsl7qjg0dlyze70kxnlzycs8v2c85kzec40gg8mt73f7y
+NEXT_PUBLIC_DEFAULT_PAIR_ADDRESS=
```

Comment:
@nguvictor could you please add the stokenet address here? to be consistent with the rest of the file